deploy/bin/deploy-app.sh

```diff
@@ -0,0 +1,5 @@
+cd "$( dirname "${BASH_SOURCE[0]}" )"/../
+
+sh ./app/bin/prepare-images.sh
```

Comment:
does it mean we prepare all 3 images every time we deploy any 1 app?

Reply:
Actually this script will be not used. To deploy app we will use script from `script` folder